apps/trustlab/src/payload/blocks/CollectionOverview.js

```diff
@@ -0,0 +1,58 @@
+function CollectionOverview(
+  slug,
+  imageURL,
+  collections,
+  {
+    fields: additionalFields = [],
+    minRows = 1,
+    maxRows = 3,
+    required = true,
+    titleLabel = "Title",
+    collectionLabel = "Resources",
+    hasLinkButton = true,
```

Comment:
`hasLink` or `hasAction` should be enough... whether it's rendered as button or not, that's a frontend issue.